config/core/build-configs.yaml

```diff
@@ -145,6 +145,25 @@ fragments:
       - 'CONFIG_REGULATOR_DA9211=y'
```

Comment:
probably is better to add this setting to the juniper device only as they are device specific not to every arm64-chromebook unless arm64-chromebook use only juniper.

Reply:
you are adding this settings that are specifics to juniper Chromebook to all the arm64-chromebook not only juniper Chromebook. is this the correct behavior?

Reply:
We could have a separate fragment just for this Chromebook, but that would mean more kernel builds.  If these extra config options don't have any side-effects on the other arm64 Chromebooks then it's fine.  The only issue is that with every additional config option to the fragment, the arm64 Chromebook build becomes a bit less like the standard mainline build.  Ideally these should be merged in the arm64 defconfig upstream, but that's not always possible.